cli/src/install.sh

```diff
@@ -1,31 +1,30 @@
-#!/bin/bash
+#!/bin/sh
 
 RED='\033[0;31m'
 GREEN='\033[0;32m'
 NC='\033[0m'
 
-# Don't continue after failure:
-set -euo pipefail
+# Fail when trying to expand unset variables
+set -u
```

Comment:
Sorry if my comment wasn't clear. We still need the "set -e" here. Just not in the command substituon "$(...)" elsewhere in the flie.


```suggestion
set -eu
```